docs/conf.py

```diff
@@ -98,6 +98,7 @@
 }
 autodoc_preserve_defaults = True
 autodoc_inherit_docstrings = False
+autodoc_typehints = "signature"
```

Comment:
This should prevent the annotations being rendered in the "Parameters" section in the description, rendering them only in the function signature.

I think it's still useful to remove them from the code, since that's the source of truth I trust